kolibri/plugins/device_management/assets/src/views/available-channels-page/index.vue

```diff
@@ -6,34 +6,36 @@
       :errorType="wizardStatus"
     />
 
-    <section
-      v-if="channelsAreAvailable"
-      class="top-matter"
-    >
-      <div class="channels">
-        <h1 class="channels-header">
-          <span v-if="inExportMode">{{ $tr('yourChannels') }}</span>
-          <span v-else-if="inLocalImportMode">{{ selectedDrive.name }}</span>
-          <span v-else>{{ $tr('channels') }}</span>
-        </h1>
-
-        <p>{{ $tr('channelsAvailable', { channels: availableChannels.length }) }}</p>
-      </div>
+    <h1>
+      <span v-if="inExportMode">{{ $tr('yourChannels') }}</span>
+      <span v-else-if="inLocalImportMode">{{ selectedDrive.name }}</span>
+      <span v-else>{{ $tr('channels') }}</span>
+    </h1>
 
-      <div class="filters">
+    <k-grid v-if="channelsAreAvailable" class="top-matter">
+      <k-grid-item sizes="4, 8, 4">
```

Comment:
In this example, the responsive grid sizes are specified as a number of columns for each breakpoint range (small, medium, and large). The key thing to remember is that small windows have 4 columns, medium have 8, and large have 12. So these three numbers can be interpreted as:

* At small sizes, this takes 4/4 columns (full-width)
* At medium sizes, this takes 8/8 columns (full-width)
* At large sizes, this takes 4/12 columns (quarter-width)

The same sizes could be specified (perhaps more intelligibly) as percentages:

```html
<k-grid-item sizes="100, 100, 25" percentage>
```